### ISSUE TITLE
More readable error log with pre-line for browser-test

### ISSUE THREAD

**Issue by taehee-sp** [thheller/shadow-cljs#1173]
## Proposal

It would be nice to add just one line css. `.error-message { white-space: pre-line; }`

[MDN : white-space](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space)
<img width="385" alt="image" src="https://github.com/thheller/shadow-cljs/assets/160077355/a38ec604-7f11-4a40-8d74-32c5fdb426ad">

I find the file. I will create Pull Request for it soon.

https://github.com/thheller/shadow-cljs/blob/a496c6febd36a7e9e6919180796f9ace81ff98e0/src/main/shadow/cljs/devtools/server/web.clj#L181

## Problem

Error message is not readable. New lines are ignored.

<img width="783" alt="image" src="https://github.com/thheller/shadow-cljs/assets/160077355/f3e0ced8-90d8-42bd-88dd-55e3eb6fcd9d">

**Comment by taehee-sp** [comment]
Is it https://github.com/bhauman/cljs-test-display 's issue?

**Comment by taehee-sp** [comment]
here!

https://github.com/bhauman/cljs-test-display/blob/master/resources/public/com/bhauman/cljs-test-display/css/style.css#L213C1-L216C2

But it seems unmaintained for 4years. 😢

**Comment by thheller** [comment]
Yes, `cljs-test-display` is the library currently used for this. I do have plans to change that though.

In the meantime you can just add the style needed to your `index.html` via a regular `<style>` tag. Or you can replace the actual .css file by making a copy and putting it into your classpath. If you already have a `resources` folder you can put it into `resources/public/com/bhauman/cljs-test-display/css/style.css`, or if you only have something like a `src/test` that would be `src/test/public/com/bhauman/cljs-test-display/css/style.css`. That should override the file from the library.
